scripts/process_image_dir.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
```

Comment:
The filename is not at all descriptive. How about upload_observation.py?

Reply:
I actually changed it from that name. :)  It includes the cleanup in the script, which is also compressing fits files, making timelapse, etc. so it is doing more than just uploading. But agree it's not the best name whereas just uploading is pretty obvious.

Reply:
Perhaps have the cleaning as an option controlled by flag(s), with uploading as the default behavior.

Reply:
The primary purpose of this script is to cleanup the directory. The uploading (and sending of metadata) are the optional arguments to that.